### PR TITLE
FEC-10188 Add more info for UNEXPECTED and SOURCE_ERROR

### PR DESCRIPTION
**Pull request by GouravSna**

### CODE REVIEW

File: playkit/src/main/java/com/kaltura/playkit/player/ExoPlayerWrapper.java [hunk 1]
```diff
@@ -671,6 +672,22 @@ private String getDecoderInitializationErrorMessage(ExoPlaybackException error,
         return errorMessage;
     }
 
+    private String getUnexpectedErrorMessage(ExoPlaybackException error, String errorMessage) {
+        Exception cause = error.getUnexpectedException();
+        if (cause.getCause() != null) {
```
Comment (giladna) [inline playkit/src/main/java/com/kaltura/playkit/player/ExoPlayerWrapper.java:678]: plz check cause is not null

File: playkit/src/main/java/com/kaltura/playkit/player/ExoPlayerWrapper.java [hunk 2]
```diff
@@ -671,6 +672,22 @@ private String getDecoderInitializationErrorMessage(ExoPlaybackException error,
...
+            errorMessage = cause.getCause().getMessage();
+        }
+        return errorMessage;
+    }
+
+    private String getSourceErrorMessage(ExoPlaybackException error, String errorMessage) {
+        Exception cause = error.getSourceException();
+        if (cause.getCause() != null) {
```
Comment (giladna) [inline playkit/src/main/java/com/kaltura/playkit/player/ExoPlayerWrapper.java:686]: also here and if error id null do not over ride the error with null plz

File: playkit/src/main/java/com/kaltura/playkit/player/ExoPlayerWrapper.java (hunk 1 above)
Comment (GouravSna) [reply playkit/src/main/java/com/kaltura/playkit/player/ExoPlayerWrapper.java:678]: cause is nonnull in Exoplayer

File: playkit/src/main/java/com/kaltura/playkit/player/ExoPlayerWrapper.java (hunk 2 above)
Comment (GouravSna) [reply playkit/src/main/java/com/kaltura/playkit/player/ExoPlayerWrapper.java:686]: We have this check in Line: 643

**Review by giladna — APPROVED** [review]

**Pull request merged**